files/mods.json

```diff
@@ -291,7 +291,7 @@
         "display": "SkyClient Updater",
         "enabled": true,
         "creator": "Lily",
-        "description": "Adds in-game updater for the mods in SkyClient.",
+        "description": "Automatically update your mods when you launch",
```

Comment:
i dislike this because it implies that updates happen behind the scenes and the user has to do nothing. on the contrary you spend like 45 seconds launching, get the update prompt, and restart your game for another 45 seconds

Reply:
not really
it gives you the option to update your mods, and restarting is optional
should i drop the "automatically"

Reply:
i mean its fine as is. i don't think anyone is going to read that and feel scammed when it doesn't turn out to be fully automatic anyways. up to you